src/registry_schemas/example_data/schema_data.py

```diff
@@ -1843,6 +1843,25 @@
     }
 }
 
+DISSOLUTION_FILING_TEMPLATE = {
+    'filing': {
+        'header': {
+            'name': 'dissolution',
+            'date': '2021-10-19',
+            'certifiedBy': 'full name',
+            'email': 'no_one@never.get',
+            'filingId': 1
+        },
+        'business': {
+            'foundingDate': '2018-01-01T00:00:00+00:00',
+            'identifier': 'CP1234567',
+            'legalName': 'legal name - CP1234567',
+            'legalType': 'CP'
+        },
+        'dissolution': DISSOLUTION
```

Comment:
so for Corps this filing template would work.  We will need to add `specialResolution` for Coops.  Not sure how we want to do this.  do we just want to add `specialResolution` and the test can delete `specialResolution` if it's a corp?

Reply:
Closing this PR. As @thorwolpert suggested, I will build the filing using base parts in the report test itself. @argush3 If we are building the filing in the test using parts, we can have different tests for different entity types that would validate the presence of special resolution for a coop. But I dont see a schema rule enforcing special resolution in a dissolution filing for coop.